<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGraph</class>
 <widget class="QWidget" name="NewGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New graph</string>
  </property>
  <widget class="QTableWidget" name="AdTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>501</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>40</horstretch>
     <verstretch>200</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewNode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="Proceed">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>100</horstretch>
     <verstretch>200</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddNode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add node</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
